python/ray/tests/test_placement_group.py

```diff
@@ -415,6 +415,9 @@ def value(self):
     result = ray.util.placement_group_table(placement_group)
     assert result["state"] == "CREATED"
 
+    # Test get all placement group.
```

Comment:
Can you test it with multiple placement group as well (not just one)?

Reply:
yes~ I will add more placement group for test.